recipes/slda/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "slda" %}
+{% set version = "0.1.6" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://github.com/Savvysherpa/slda/archive/{{ version }}.tar.gz
+    sha256: aeabfe4edefc566ee97c107f8f9b2c6a478595e24fb35d44d9eeaaa876fb90ce
+
+build:
+    number: 0
+    skip: True  # [win or not py3k]
+
+requirements:
+    build:
+        - cython
+        - cythongsl
```

Comment:
> cythongsl depends on gsl. To avoid over-specifying the dependencies you can remove gsl from both build and run sections.

[quote](https://github.com/conda-forge/staged-recipes/pull/1018#discussion-diff-70455031) from @ocefpaf